app/src/main/java/net/skyscanner/backpack/demo/compose/BadgeStory.kt

```diff
@@ -95,6 +97,14 @@ private fun BadgeRow(
                 else -> BpkIcon.TickCircle
             },
         )
+
+        BpkBadge(
```

Comment:
please regenerate the docs screenshots as well

Reply:
updated